src/Controls/src/Core/HybridWebView/PlatformHybridWebViewWebResourceRequestedEventArgs.cs

```diff
@@ -53,50 +56,13 @@ internal PlatformHybridWebViewWebResourceRequestedEventArgs(WebResourceRequested
 
 	internal string? GetRequestMethod() => Request.Method;
 
-	internal IReadOnlyDictionary<string, string>? GetRequestHeaders() => new WrappedHeadersDictionary(Request.Headers);
-
-	class WrappedHeadersDictionary : IReadOnlyDictionary<string, string>
```

Comment:
The previous way to use wrappers was me being "so clever" and "avoiding allocations" and "reducing copies"... But yeah, that was not sensible since the amount of things that would happen was so little. In order to hit this path, the user has to actually request the headers. 

For all other cases, this doesn't do anything.